test/plugin_functional/test_suites/core_plugins/rendering.ts

```diff
@@ -230,6 +230,7 @@ export default function ({ getService }: PluginFunctionalProviderContext) {
         'xpack.cloud.projects_url (any)', // It's a string (any because schema.conditional)
         // can't be used to infer urls or customer id from the outside
         'xpack.cloud.serverless.project_id (string)',
+        'xpack.cloud.serverless.project_name (string)',
```

Comment:
@elastic/kibana-security I don't think this is a problem given it's only exposed on authenticated page and given we're going to expose the info in the Kibana navigation header, but still pointing this out.

Reply:
Sounds good, considering that we're already exposing `project_id` which is more important.

Just curious, if we ever allow user to rename the project, does it mean we'll have to restart Kibana to apply the config change? Feels a bit heavy.

Reply:
The way the project controller works, changing the name from the Cloud UI will trigger an update of the deployment, yes. But there's isn't really ways around this until Cloud exposes an API for Kibana to get the info less passively